webapp/src/ts/components/actionbar/actionbar.component.ts

```diff
@@ -48,6 +48,7 @@ export class ActionbarComponent implements OnInit, OnDestroy {
       this.store.select(Selectors.getSelectMode),
       this.store.select(Selectors.getShowActionBar),
       this.store.select(Selectors.getSelectedReportsDocs),
+      this.store.select(Selectors.getSelectedContact),
```

Comment:
above comment reffers to this selector not being `selectedContactDoc` like in https://github.com/medic/cht-core/blob/master/webapp/src/js/selectors/index.js#L38

Reply:
I've switched it for selectedContactDoc selector